### DISCUSSION TITLE
Switching character sheet from Zweihander to Flames of Freedom

### DISCUSSION THREAD

**Discussion by Aberosk** [fh-fvtt/zweihander#190] [Q&A]
First off thank you and holy cow this looks great in Foundry! I am currently about to initiate a Flames of Freedom campaign and cannot get, or rather do not understand how to change character sheets from Zweihander sheets to FoF. Any help would be appreciated!

**Comment by Re4XN** [comment]
You can do it in system settings:

![image](https://user-images.githubusercontent.com/16267371/223704383-c304e694-75d0-44aa-82f5-6d88e4b4bcbe.png)

**Reply by Aberosk** [reply]
I was able to do that, however there's a different type of character sheet for FoF that tracks 'Flaw ----- Conflict ----- My Belief Is...' rather than "Order Alignment ........... Corruption ............ Chaos Alignment" I was hoping to adjudicate that change.
